src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -416,3 +418,25 @@ export function getTileName(x: number, y: number, gridSize: GridSize): string {
   const tileId = `${`${tileY}`.padStart(nbDigits, '0')}${`${tileX}`.padStart(nbDigits, '0')}`;
   return `${sheetCode}_${gridSize}_${tileId}`;
 }
+
+/**
+ * Verify if a TIFF contains only 8 bits bands.
+ *
+ * @param tiff
+ * @returns true if 8 bits TIFF
+ */
+export async function is8BitsTiff(tiff: Tiff): Promise<boolean> {
```

Comment:
Would it make sense to change this so that we won't have to change the name once we support other bit depths? That is, would a name like `validateBitDepth` be future-proof?

Reply:
That is something I generally do when the need of supporting something else comes.